content_api/auth/service.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+import bcrypt
+from flask import g, current_app as app
+from apps.auth.service import AuthService
+from superdesk import get_resource_service
+from apps.auth.errors import CredentialsAuthError
+
+from superdesk.utc import utcnow
+from datetime import timedelta
+
+
+class ContentApiAuthService(AuthService):
```

Comment:
I'm not sure if we need such service, users should be authenticated using sessions, and for api we should use some tokens that users could generate, but wouldn't authenticate on the api with email/password